quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testExportHelp.unix.approved.txt

```diff
@@ -10,9 +10,27 @@ Options:
 
 -h, --help           This help message.
 --help-all           This same help message but with additional options.
+--optimized          Use this option to achieve an optimal startup time if you have previously
+                       built a server image using the 'build' command.
+
+Cache:
```

Comment:
Do cache options make sense when running export/import?

Reply:
The cache is enabled/started when running export/import, therefore it should be configurable, therefore I want to keep it.

Reply:
Could you elaborate on why it should be configurable? Why not just make it `local` as it works today?

Reply:
What I'm proposing is to:

* Update the import/export profile (see https://github.com/keycloak/keycloak/blob/main/quarkus/runtime/src/main/resources/META-INF/keycloak.conf)
* Remove cache options from CLI
* Optionally, always resolve `cache=local` if running export/import. To avoid overriding this via other config sources.